nchelpers/__init__.py

```diff
@@ -725,7 +725,7 @@ def is_multi_year_mean(self):
         # Additional non-strict heuristics begin here
 
         # Heuristic: Time variable has "suspicious" length:
-        # 1, 4, 12, 5, 13, 16, 17 (yearly, seasonal, monthly, and
+        # 1, 4, 12, 365, 5, 13, 16, 17 (yearly, seasonal, monthly, daily, and
```

Comment:
Do we ever encounter 360-day years? (Or, god forbid, standard calendars with 366-day years every 4th?)

Reply:
* Will there ever be 366-timestamp climatologies?
I am reasonably confident the answer on this is "no". 

As you know, a climatological dataset has a variable called `climatology_bnds` which has dimensions `2 * [number of timestamps in the climatology]` and records the range of dates represented by each timestamp in the climatology. 

So for a daily climatology that runs 1971 - 2000, the timestamps would be January 1 1985, January 2 1985 etc, and the values in climatology_bnds would be [January 1 1971, January 1 2000], [January 2 1971, January 2 2000], etc to indicate that the first timestamps represents all January 1 from 1971 to 2000, the second timestamps represents all January 2 from 1971 to 2000. These climatology bounds are added to the database in the `climatological_times` table.

The climatological_times table uses postgres date objects. So it can only record days that actually exist. This is relevant because if we have a 366-timestamp climatology, we would have an entry for February 29th, which would need to be recorded in climatology_bnds and in the database as February 29 1971 and February 29 2000. The latter exists, but the former does not, and cannot be recorded in the database with a postgresql date object. 

And since we use 30 year climatologies, and 30 is not divisible by 4, it will _never_ be the case that we can record a daily climatology with an entry for February 29th.

(In fact, the data I am trying to add to the database now uses 366-day years, and I had to remove February 29th from all the datasets before I could index them.)


* Will there ever be 360 day climatologies?
This seems, regrettably, possible. I will add checks for it.